lib/kitchen/directions/bake_numbered_table/v1.rb

```diff
@@ -4,6 +4,7 @@ module Kitchen::Directions::BakeNumberedTable
   class V1
 
     def bake(table:, number:, always_caption: false)
+      table.remove_attribute('summary')
```

Comment:
[in this PR](https://github.com/openstax/kitchen/pull/249) I thought I had removed the summary attr BUT it was still there but with different data